packages/react-native-gesture-handler/src/v3/hooks/useGesture.ts

```diff
@@ -81,20 +82,21 @@ export function useGesture<THandlerData, TConfig>(
   ) {
     currentGestureRef.current = { type, tag };
     RNGestureHandlerModule.createGestureHandler(type, tag, {});
-    RNGestureHandlerModule.flushOperations();
+    // It's possible that this can cause errors about handler not being created when attemting to mound NativeDetector
```

Comment:
```suggestion
    // It's possible that this can cause errors about handler not being created when attempting to mount NativeDetector
```

Reply:
I don't know - it cannot be the cause, since it wasn't there when you found it, but I also was able to reproduce that only by modifying RNGH source while the app was running.